testapi.pm

```diff
@@ -1012,6 +1012,12 @@ sub background_script_run {    # no:style:signatures
     return $distri->background_script_run($cmd, %args);
 }
 
+sub _set_assert_marker {
+    my ($hashed_string) = @_;
```

Comment:
For any new functions please use signatures, i.e.:

```suggestion
sub _set_assert_marker ($hashed_string) {
```

Reply:
*ready for review*